### PR TITLE
Request to merge List interaction

### PR DESCRIPTION
**Pull request by Kingstalux**
I added some functionality to the application to make it interactive. 
The user will be able to mark task completion by selecting the corresponding checkbox (or undo it by unchecking the checkbox). The updated tasks list will be stored in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -31,13 +38,33 @@ tasks.forEach((element) => {
   div.appendChild(div2);
   const checkbox = document.createElement('input');
   checkbox.type = 'checkbox';
+  checkbox.className = 'check';
   div2.appendChild(checkbox);
   const para = document.createElement('p');
   para.innerText = element.description;
   div2.appendChild(para);
+  checkbox.addEventListener('click', () => {
```
Comment (eliasecasta) [inline src/index.js]: - [x] You used a click listener but it is required for this project to use an on-change listener, please switch to that one in order to comply with the project requirements.

File: src/index.js [hunk 2]
```diff
@@ -1,27 +1,34 @@
...
   {
     description: 'wash the car',
     completed: false,
-    index: 3,
+    index: 2,
   },
 ];
 
-tasks.forEach((element) => {
+let taskList = JSON.parse(localStorage.getItem('tasks'));
```
Comment (eliasecasta) [inline src/index.js:25]: - [x] I am unable to see any change in the local storage variables, before or after refreshing the page. Please take a look at this bug and solve it to comply with the project requirements.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @Kingstalux, I am Elias, your reviewer for this PR

Well done so far with your project!

This project is almost complete, you just have to solve some really small issues first.

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [ ] Please check the comments below.

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (Kingstalux) [reply src/index.js]: Okay, I've applied this.

File: src/index.js (hunk 2 above)
Comment (Kingstalux) [reply src/index.js:25]: Okay, I've arranged the bug and I've uploaded two screenshots to show the changes applied to the local storage before and after a user clicks the checkbox. It also stays the same on reload
Screenshot1
Screenshot2
.

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @Kingstalux, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**